### DISCUSSION TITLE
Is there a recommended pg_partman naming convention for integer range partitioning?

### DISCUSSION THREAD

**Discussion by rgidds** [pgpartman/pg_partman#629] [Q&A] [answered]
Hello,

No issues, just a question, Is there a recommended pg_partman naming convention for integer range partitioning?

Thank you,
Rachel

**Comment by keithf4** [comment, marked as answer]
Moving this to a discussion.

The only real conventions around partitioning are the suffixes and with pg_partman, that is automatically maintained for you. My only suggestion is to keep your table name as short as you can since the suffix will continue to grow longer over time and Postgres has a 63 byte (not character) limit on object names. Partman also manages that for you, truncating the table name before adding the suffix, but it can be startling to suddenly see your table names getting cut off.

**Comment by rgidds** [comment]
Thank you for the suggestion, appreciate it.
